### PR TITLE
fix issue with refresh_schema thread

### PR DESCRIPTION
**Pull request by tanjabrzak**
https://github.com/lumoslabs/aleph/issues/103

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,10 @@ Change Log
 
 All notable changes to this project will be documented in this file using [Semantic Versioning](http://semver.org/).
 
+## [0.4.8] - 2020-01-20
+### Fixed
```
Comment (joycelau1) [inline CHANGELOG.md]: @tanjabrzak please create an [issue](https://github.com/lumoslabs/aleph/issues) with a description the problem and add the link here like [this](https://github.com/lumoslabs/aleph/blob/fa4d01cc37ae9da01eed0d5f8cfc1b60b1d18c2c/CHANGELOG.md#fixed-3)

File: lib/schemas/descriptor.rb [hunk 2]
```diff
@@ -61,7 +62,7 @@ def retrieve
     def schema_refresher
       loop do
         refresh_schema
-        sleep(SCHEMA_REFRESH_INTERVAL)
+        sleep(SCHEMA_REFRESH_INTERVAL.to_i)
```
Comment (PalSzabolcs) [inline lib/schemas/descriptor.rb]: hm, this shouldn't be needed, ruby automatically converts `1.hour` to 3600, but i guess there is a reason to this change?

**Review by PalSzabolcs — APPROVED** [review]
minor comment, but looks ok

File: lib/schemas/descriptor.rb (hunk 2 above)
Comment (tanjabrzak) [reply lib/schemas/descriptor.rb]: when value gets from yml, than it is parsed as string
if it gets default value, parsing to int will work too

Comment (PalSzabolcs) [reply lib/schemas/descriptor.rb:65]: yeah, strange that i won't get parsed as an integer, but this should than fix it!

**Pull request merged**